docs/client/apidocs.rst

```diff
@@ -0,0 +1,110 @@
+API Reference
+=============
+
+Module contents
+---------------
+
+.. automodule:: scrapinghub.client
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Submodules
+----------
+
+activity
+--------
+
+.. automodule:: scrapinghub.client.activity
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+collections
+-----------
+
+.. automodule:: scrapinghub.client.collections
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
In case of collectoins it shows `Bases: scrapinghub.client.utils._Proxy`, not clickable, not documented - not useful.